docs/streams/quickstart.html

```diff
@@ -140,25 +130,36 @@
     --topic streams-file-input
 </pre>
 
+The created topic can be described with the same <b>kafka-topics</b>> tool:
 
 <pre class="brush: bash;">
-&gt; bin/kafka-console-producer.sh --broker-list localhost:9092 --topic streams-file-input < file-input.txt
+&gt; bin/kafka-topics.sh --zookeeper localhost:2181 --describe
```

Comment:
@guozhangwang yes, the deprecation is only for changing topic configs via the `kafka-topics` command. I personally think we should undeprecate it, but that's where we are today. All the other `kafka-topics` functionality is not deprecated.